src/react-components/info-dialog.js

```diff
@@ -214,6 +217,10 @@ class InfoDialog extends Component {
           </div>
         );
         break;
+      case InfoDialog.dialogTypes.link:
+        dialogTitle = "Send Link to Device";
```

Comment:
This title might be a bit confusing to a user. What "link" are you referring to? Maybe It should just be "Enter on Device".

Reply:
Ah yes I had that copy previously, the reason I put it here is because I'm trying to help the user memorize `hub.link` -- it's a stretch